spec/features/comments_spec.rb

```diff
@@ -12,18 +12,35 @@
 
   describe 'for a normal guide' do
     it 'write a comment successfully' do
-      write_a_comment_successfully(guide: guide)
+      write_a_comment_successfully(
+        guide: guide,
+        comment: "This is a guide comment",
+      )
+    end
+
+    it "auto links urls in comments" do
+      guide = create(:guide)
+      write_a_comment_successfully(
+        guide: guide,
+        comment: "This is a link: http://google.com",
+      )
+      within ".comments .comment" do
+        expect(page).to have_link "http://google.com", href: "http://google.com"
+      end
```

Comment:
Nicely written test.

Maybe we could delete the one above? It seems to be covered by this one.